src/dbSta/src/dbSta.tcl

```diff
@@ -93,7 +93,7 @@ proc report_cell_usage { args } {
 
   set module [[ord::get_db_block] getTopModule]
   if { $args != "" } {
-    set modinst [[ord::get_db_block] findModInst $args]
+    set modinst [[ord::get_db_block] findModInst [lindex $args 0]]
```

Comment:
The command doesn't take a list.  It might be surprising to someone who tries that to get only the first element.  Is an error better?

Reply:
we already check for this: `  check_argc_eq0or1 "report_cell_usage" $args` above (`args` must be length 0 or 1)

Reply:
Not really:
```
>>> report_cell_usage [list 1 2 3]
[ERROR STA-1002] Unable to find {1 2 3}
[ERROR GUI-0070] STA-1002
>>> report_cell_usage  1 2 3
[ERROR STA-0566] report_cell_usage requires zero or one positional arguments.
[ERROR GUI-0070] STA-0566
```

Reply:
in the first case, you are passing in an list, which gets converted to "{1 2 3}", not sure what we would check then, this is true for everything, if you pass in a list when a list is not expected it's wrong.